core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -0,0 +1,226 @@
+package decryption_queue
```

Comment:
Package name is normally aligned with the directory (besides `package main`s):
```suggestion
package threshold
```
And then you could consider dropping any redundant `Threshold` prefixes from the identifiers, since they will be accessed like:
```diff
-threshold.ThresholdDecryptor
+threshold.Decryptor

-threshold.NewThresholdDecryptor
+threshold.NewDecryptor
```

Reply:
Thanks!
This package is intended to be used by the Threshold Decryption plugin which is currently being built by Research.  I believe a more optimal file structure might be for me to create a subdirectory within `/threshold` called `/decryption_queue`.

Reply:
Hmmm go package are not usually scoped so narrowly. It leads to confusing naming patterns like `decryption_queue.DecryptionQueue`. Where will the other code be located? A name collision on `package threshold` from two different modules is usually no big deal, since the imports can be aliased when necessary.

Reply:
I believe the plugin code will be contained within the `/threshold` directory.  Therefore, we could just include the `decryption_queue` code and the plugin code in a single `package threshold`.